.github/workflows/cibuildwheel.yaml

```diff
@@ -199,6 +212,26 @@ jobs:
         run: |
           tools/check_dist.py ${{ github.event_name == 'release' && format('-r {0}', github.ref) || '' }}
 
+  publish-wheels:
+    name: publish-wheels
+    needs:
+      - build-wheels
+      - check-wheels
+      - publish-sdist
+    runs-on: ubuntu-latest
+
+    permissions:
+      # NOTE(vytas): This permission is mandatory for Trusted Publishing.
```

Comment:
Maybe, also mention that no other/untrusted actions are allowed to be added into this job (in case people get ideas over time).